src/components/Assemblies/SeatChart.svelte

```diff
@@ -126,3 +144,14 @@
 		class="transition-[height] duration-500 ease-in-out"
 	/>
 </div>
+
+<div
+	style:left={`${tooltipProp?.x ?? 0}px`}
+	style:top={`${tooltipProp?.y ?? 0}px`}
+	class:hidden={!tooltipProp}
+	class="label-01 absolute z-10 flex min-w-max flex-col
+		rounded-sm bg-[#393939] p-3 text-center text-white"
+>
+	<div class="font-semibold">{tooltipProp?.title}</div>
+	<div>{tooltipProp?.additional}</div>
+</div>
```

Comment:
Same comment as above and looks like the code is exactly the same. It would be great if we move this template to a new component in the same folder, receiving `tooltipProp` as a prop, so we don't have a duplicated code doing exactly the same thing.

Reply:
Moved to AssemblyTooltip component.